about.html

```diff
@@ -37,7 +37,7 @@
           <a href="">&times;</a>
           <a href="index.html" class="close-menu">Home</a>
           <a href="about.html" class="close-menu">About Us</a>
-          <a href="#" class="close-menu">Inquirires</a>
+          <a href="" class="close-menu">Inquirires</a>
           <a href="#" class="close-menu">Partner</a>
        </div>
```

Comment:
- [ ] In this section 👇🏼, kindly consider adding a section header, a divider, and a line of text before the two images with text centered over them, so this section looks exactly as the provided design

|Your website|Provided design|
|-|-|
|![Screenshot_20230706_202648](https://github.com/paulshaun-kdtk/capstone-Car-Show/assets/25227022/38dc056b-cf8d-4635-9755-43b8f41f4548)|![Screenshot_20230706_202705](https://github.com/paulshaun-kdtk/capstone-Car-Show/assets/25227022/37698f23-8c36-4647-b476-d6841c7693e7)|